### PR TITLE
example on basic usage

### PR DESCRIPTION
**Pull request by omichels**
i just added an example of a basic usage

### CODE REVIEW

File: examples/basic/provider.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      source  = "hashicorp/azurerm"
+      version = "~>3.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  # https://github.com/hashicorp/terraform-provider-azurerm/issues/3358
+  # our SP cannot register surplus azure providers for like Media or AI or chat-bots
+  skip_provider_registration = true
```
Comment (dploeger) [inline examples/basic/provider.tf]: Could you please remove this, because it seems to be pretty special for your use case. Thanks!

Comment (dploeger) [inline examples/basic/provider.tf]:
```suggestion
```

**Review by dploeger — CHANGES_REQUESTED** [review]
Thanks, Oliver! Just a small remark.

File: examples/basic/provider.tf (hunk 1 above)
Comment (omichels) [reply examples/basic/provider.tf]: done

Comment (omichels) [reply examples/basic/provider.tf]: done

**Comment by omichels** [comment]
greetings back to one of my favourite cities in the Ruhr valley: Dortmund

**Comment by dploeger** [comment]
Oh sorry, just saw that you didn't comply with our other commits. Could you please rebase and use something like "chore: Added example" as the commit message?

**Comment by omichels** [comment]
rebased and forced-pushed commits do not show up, when an open PR exists. 
Closing this PR therefore

**Pull request closed without merging**